style.css

```diff
@@ -579,3 +621,317 @@ label {
+  }
+
+  .nav-links {
+    padding: 8px;
+    margin-right: 670px;
+  }
+
+  .projects {
+    display: grid;
+    grid-template-columns: 90%;
+    margin-top: 80px;
+    max-width: 100%;
+    column-gap: 30px;
+  }
+
+  .work {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    background-color: #fff;
+  }
+
+  .lost {
+    display: none;
+  }
+
+  .desktop-image {
+    display: inline;
+    width: 100%;
+    padding: 10px;
+    margin-right: 190px;
+  }
+
+  .work-section h2 {
+    font-size: 40px;
+    width: 450px;
+    position: relative;
+    right: 50px;
+    bottom: 15px;
+  }
+
+  .about-paragraph {
+    width: 80%;
+    font-size: 16px;
+  }
+
+  .labels h2 {
+    font-size: 40px;
+  }
+
+  .work-load .canopy {
+    font-size: 18px;
+  }
+
+  .work-load .back-end {
+    font-size: 18px;
+    width: 130px;
+    font-weight: 400;
+  }
+
+  .work-load .year {
+    font-size: 18px;
+    font-weight: 400;
+  }
+
+  .canopy-1,
+  .back-1,
+  .paragraph-1 {
+    display: none;
+  }
+
+  .desktop-paragraph,
+  .facebook,
+  .front-end {
+    display: inline;
+  }
+
+  .facebook2,
+  .front-end2,
+  .desktop-paragraph2,
+  .header-facebook {
+    display: inline;
+  }
+
+  .labels-3 h2 {
+    font-size: 40px;
+    font-weight: 700;
+  }
+
+  .work-three .facebook2 {
+    font-size: 18px;
+    font-family: 'Poppins', sans-serif;
+    line-height: 16px;
+    font-weight: 600;
+    font-style: normal;
+    color: #344563;
+  }
+
+  .work-three .front-end2 {
+    font-size: 18px;
+    color: #7a869a;
+    line-height: 16px;
+    font-weight: 400;
+    font-style: normal;
+    font-family: 'Poppins', sans-serif;
+  }
+
+  .work-three .year {
+    font-size: 18px;
+    font-weight: 400;
+  }
+
+  .desktop-paragraph2 {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .labels-3 .project-button {
+    background-color: #4053fc;
+    color: #fff;
+  }
+
+  .headerz,
+  .uber,
+  .year-1,
+  .desktop-paragraph3,
+  .lead-dev {
+    display: inline;
+  }
+
+  .labels-4 h2 {
+    font-size: 40px;
+  }
+
+  .work-four .uber {
+    font-size: 19px;
+    font-family: 'Poppins', sans-serif;
+    line-height: 16px;
+    font-weight: 600;
+    font-style: normal;
+    color: #344563;
+  }
+
+  .work-four .lead-dev,
+  .work-four .year-1 {
+    font-size: 18px;
+    color: #7a869a;
+    line-height: 16px;
+    font-weight: 400;
+    font-style: normal;
+    font-family: 'Poppins', sans-serif;
+  }
+
+  .desktop-paragraph3 {
+    font-size: 16px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .move {
+    margin-left: 10px;
+  }
+
+  .about-me {
+    width: 100%;
+    display: grid;
+    grid-template-columns: 50% 50%;
+    justify-content: space-evenly;
+    height: auto;
+    margin-top: 10vh;
+    border-top-right-radius: 100px;
+  }
```

Comment:
- [ ] Kindly make sure each project card looks similar to the given [Figma design 🎨](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=aBYhrF8gMscZ1eny-0) as I've noticed some spacing issues on your "About me" section design:
    - Your design:
      ![firefox_AGaPEgSgRy](https://user-images.githubusercontent.com/104968236/228873934-df5892cc-d41b-44d5-a3be-0b47af2220ec.png)
    - Figma design:
      ![InkedInkedfirefox_pdNdcNysBC](https://user-images.githubusercontent.com/104968236/228874939-a25c929b-4d5e-4bc4-8587-1255ba0ab3d5.jpg)
- As I've marked above your design seems to be missing some padding. So please fix that as shown in the design. ✔